### PR TITLE
Aarch64 no gzip

### PR DESCRIPTION
**Pull request by gc00**
We have a long-standing bug for DMTCP with --gzip under aarch64.  Until we find the time to look at it carefully, we should automatically turn off --gzip.  This patch does that for aarch64..

### CODE REVIEW

File: src/dmtcp_launch.cpp [hunk 1]
```diff
@@ -330,6 +331,23 @@ static void processArgs(int *orig_argc, char ***orig_argv,
       break;
     }
   }
+#define __aarch64__ 1
```
Comment (karya0) [inline src/dmtcp_launch.cpp]: That doesn't look correct (a leftover from testing?) :)

**Comment by gc00** [comment]
Thanks, @karya0.  You're right.  I had a leftover line of code from testing.

I believe it's fixed now.  Could you re-review it?

**Comment by rohgarg** [comment]
Looks good to me.

**Comment by rohgarg** [comment]
Looks good to me.

**Pull request merged**